core/templates/dev/head/services/local-storage.service.spec.ts

```diff
@@ -53,6 +53,10 @@ describe('LocalStorageService', () => {
         draftDictTwo);
     });
 
+    it('should storage be available', () => {
```

Comment:
`'should storage be available'` . -> `'should verify that storage be available'`

Reply:
Done.